server/api.go

```diff
@@ -334,6 +335,10 @@ func DoResultCallback(callbackUrl string, result *SessionResult, issuer string,
 
 	var x string // dummy for the server's return value that we don't care about
 	if err := irma.NewHTTPTransport(callbackUrl, false).Post("", &x, res); err != nil {
+		if sessErr, ok := err.(*irma.SessionError); ok && sessErr.RemoteStatus == http.StatusNoContent {
+			// If we get a 204 during this callback that's perfectly fine
+			return
+		}
```

Comment:
Since we ignore the result `x` here, I think it should also work to remove `x` here and say `Post("", nil, res)`. This means we ignore the result. `Post` is implemented in such a way that if a 204 is received and the result is nil, then no error is returned. This means we wouldn't have to manually catch this error here.

Reply:
You're right. I'll change that.

Reply:
Yeah it seems logical behaviour to me to also add 204 support for the normal callback option then indeed. Good catch.